frontend/src/plugin/registry.tsx

```diff
@@ -623,4 +624,26 @@ export function registerHeadlampEventCallback(callback: HeadlampEventCallback) {
   store.dispatch(addEventCallback(callback));
 }
 
+/**
+ * Register a plugin settings component.
+ * @param name - The name of the plugin.
+ * @param component - The component to use for the settings.
+ * @param displaySaveButton - Whether to display the save button.
+ *
+ * @example
+ *
+ * ```tsx
+ * import { registerPluginSettings } from '@kinvolk/headlamp-plugin/lib';
+ *
+ * registerPluginSettings('my-plugin', MyPluginSettingsComponent, true);
+ * ```
```

Comment:
Can you please also mention the examples where this function is used?

Similar to this part of the comment for another function in registry.tsx.
```
 * More complete logo example in plugins/examples/change-logo:
 * @see {@link http://github.com/kinvolk/headlamp/plugins/examples/change-logo/ Change Logo Example}
```